apps/voting/app/src/utils.js

```diff
@@ -8,3 +10,7 @@ export function pluralize(count, singular, plural, re = PLURALIZE_RE) {
 }
 
 export function noop() {}
+
+export function formatDate(date) {
+  return `${format(date, 'dd/MM/yy')} at ${format(date, 'HH:mm')} UTC`
```

Comment:
@dizzypaty This produces strings like:

```
07/12/18 at 16:22 UTC
```

But perhaps a more natural sounding alternative:

```
16:22 UTC on 07/12/18
```

Or alternatively one that spells out the month and day, since `07/12` is ambiguous across regions (July 12, or Dec. 7)?

Reply:
Yes, I think it's clearer if we use this format (if possible): 

`16:22 UTC on 12th of July, 2019`

Reply:
How should we treat this:
![image](https://user-images.githubusercontent.com/3072458/63015155-86581b80-be90-11e9-9ded-c92529743753.png)

Reply:
Going for this format:
![image](https://user-images.githubusercontent.com/3072458/63015317-e18a0e00-be90-11e9-8409-0bc7be45431a.png)